bin/base

```diff
@@ -218,6 +225,7 @@ function start_valkey() {
 function stop_valkey() {
     local RC=0
     echo -n "Checking for valkey"
+    rm -f /tmp/valkey_pass
```

Comment:
This one is less important, but it seems like it would be "cleaner" to put this inside the if statement also.